packages/manager/apps/web-office/src/pages/dashboard/users/ModalOrderLicenses.component.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useContext, useMemo } from 'react';
+import {
+  OdsFormField,
+  OdsMessage,
+  OdsSelect,
+  OdsQuantity,
+} from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+import { getExpressOrderURL } from '@ovh-ux/manager-module-order';
+import { ODS_BUTTON_VARIANT } from '@ovhcloud/ods-components';
+import { OvhSubsidiary } from '@ovh-ux/manager-react-components';
+import { ShellContext } from '@ovh-ux/manager-react-shell-client';
+import { useForm, Controller } from 'react-hook-form';
+import Modal from '@/components/Modals/Modal';
+import {
+  useGenerateUrl,
+  useOfficeLicenseDetail,
+  useOrderCatalog,
+} from '@/hooks';
+import { generateOrderURL } from '@/api/order';
+
+type FormData = {
+  productType: string;
+  quantity: number | null;
+};
+
+export default function ModalOrderLicenses() {
+  const { t } = useTranslation(['dashboard/users/order-licenses', 'common']);
+  const navigate = useNavigate();
+  const context = useContext(ShellContext);
+  const ovhSubsidiary = context.environment.getUser()
+    .ovhSubsidiary as OvhSubsidiary;
+  const region = context.environment.getRegion();
+  const { data: dataLicenceDetail } = useOfficeLicenseDetail();
+  const { data: catalog } = useOrderCatalog({
+    productName: 'officePrepaid',
+    ovhSubsidiary,
+  });
+  const productOptions = useMemo(
+    () =>
+      catalog?.plans
+        .map((plan) => ({
+          label: plan.invoiceName,
+          value: plan.planCode,
+        }))
+        .sort((a, b) => a.label.localeCompare(b.label)),
+    [catalog],
+  );
+
+  const orderBaseURL = getExpressOrderURL(region, ovhSubsidiary);
+  const goBackUrl = useGenerateUrl('..', 'path');
+  const onClose = () => navigate(goBackUrl);
+
+  const { handleSubmit, control, watch } = useForm<FormData>({
+    defaultValues: {
+      productType: '',
+      quantity: null,
+    },
+  });
+
+  const onSubmit = (data: FormData) => {
+    const products = [
+      {
+        planCode: data.productType,
+        quantity: data.quantity || 1,
+        serviceName: dataLicenceDetail?.tenantServiceName as string,
+      },
+    ];
+    window.open(
+      generateOrderURL({ baseURL: orderBaseURL, products }),
+      '_blank',
+    );
+  };
+
+  return (
+    <Modal
+      title={t('common:order-licenses')}
+      onClose={onClose}
+      isDismissible
+      isOpen
+      secondaryButton={{
+        label: t('common:cta_cancel'),
+        action: onClose,
+        testid: 'cancel-btn',
+        variant: ODS_BUTTON_VARIANT.outline,
+      }}
+      primaryButton={{
+        label: t('common:cta_confirm'),
+        action: handleSubmit(onSubmit),
+        testid: 'confirm-btn',
+        isDisabled: !watch('productType'),
+      }}
+    >
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <div className="flex flex-wrap">
+          <OdsFormField className="w-3/5 mr-8">
+            <label slot="label">
```

Comment:
I think you are supposed to pass a `for` prop to link it to the input (to apply to all labels)